aspnetcore/fundamentals/servers/index.md

```diff
@@ -13,20 +13,20 @@ By [Tom Dykstra](https://github.com/tdykstra), [Steve Smith](https://ardalis.com
 
 An ASP.NET Core app runs with an in-process HTTP server implementation. The server implementation listens for HTTP requests and surfaces them to the app as sets of [request features](xref:fundamentals/request-features) composed into an <xref:Microsoft.AspNetCore.Http.HttpContext>.
 
-ASP.NET Core ships with the following server implementations:
+ASP.NET Core ships with the following:
 
 ::: moniker range=">= aspnetcore-2.2"
 
-* [Kestrel](xref:fundamentals/servers/kestrel) is the default, cross-platform HTTP server for ASP.NET Core.
-* `IISHttpServer` is used with the [in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model) and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module) on Windows.
-* [HTTP.sys](xref:fundamentals/servers/httpsys) is a Windows-only HTTP server based on the [HTTP.sys kernel driver and HTTP Server API](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx). (HTTP.sys is called [WebListener](xref:fundamentals/servers/weblistener) in ASP.NET Core 1.x.)
+* [Kestrel server](xref:fundamentals/servers/kestrel) is the default, cross-platform HTTP server for ASP.NET Core.
+* IIS HTTP Server (`IISHttpServer`) is a processing layer used with the [in-process hosting model](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model) and the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module) on Windows.
```

Comment:
I would remove "processing layer" in favor of "is a server that is only used with IIS". The IIS HTTP Server is a true server implementation, it's just that the managed layer only needs to translate request from native to managed because IIS parses the request for us.

Reply:
IIS HTTP Server is a IIS-only HTTP server.